docs/datatable-tutorial/simple_table.md

```diff
@@ -0,0 +1,104 @@
+```python exec
+import reflex as rx
+from pcweb.templates.docpage import docdemobox, doclink
+from pcweb.base_state import State
+from pcweb.pages.docs.datatable_tutorial.states import DataTableState, DataTableState2
+```
+
+
+
+# Data Table Tutorial
+
+```python eval
+rx.alert(
+        rx.alert_icon(),
+        rx.alert_title(f"There is another ",
+        doclink("datatable component", href="https://reflex.dev/docs/library/datadisplay/datatable/"),
+        ", but this is only used for displaying data and there is no possible user interaction while using this component"),
+        status="info",
+    )
+```
+
+
+```python eval
+rx.box(height="2em")
+```
+We need to start by defining our `columns` that will then take in our data. These columns should be a list of dict, where each dict describes the associated columns. 
+
+Each dict has these variables below:
+1. title: The text to display in the header of the column
+2. id: An id for the column, if not defined, will default to a lower case of title
+3. width: The width of the column
+4. type: The type of the columns, default to "str"
```

Comment:
Use code marks on the key names

```suggestion
1. `title`: The text to display in the header of the column
2. `id`: An id for the column, if not defined, will default to a lower case of title
3. `width`: The width of the column (in pixels?)
4. `type`: The type of the columns, default to "str"
```

For `type`, how can i find out what types are available?

Reply:
are there more recognized keys for a column dict as well? where are those defined?

Reply:
as far as I can tell these are the only 4, but going through the reflex docs and the reflex code these are not actually defined anywhere that I can see so it is hard to say if there are more, we will need to chat to Thomas and ask